hack/build.sh

```diff
@@ -25,6 +25,11 @@ fi
 
 set -eu
 
+# Temporary fix for iTerm issue https://gitlab.com/gnachman/iterm2/issues/7901
```

Comment:
I suggest replacing the 4 lines with

```
[[ -n "${ITERM_PROFILE:-}" ]] S=" " || S=""
```

Reply:
Updated but kept the if-syntax although a bit more terse but easier to understand for non-shell gurus.